src/main/resources/application.yaml

```diff
@@ -53,13 +58,20 @@ idam:
   api:
     url: ${IDAM_API_BASEURL:https://idam-api.demo.platform.hmcts.net}
   serviceAccount:
+    clientId: ${HMC_FIS_HEARING_SERVICE_CLIENT_ID:hmc-fis-hearing-service}
+    clientSecret: ${HMC_FIS_HEARING_SERVICE_CLIENT_SECRET:dummy}
+    grantType: ${IDAM_HMC_COMPONENT_GRANT_TYPE:password}
```

Comment:
need to set it in values.yml  if these are new or not set in

Reply:
done